AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/README.md

```diff
@@ -0,0 +1,69 @@
+# Tutorial : Optimize TensorFlow pre-trained model for inference
+This tutorial will guide you how to optimize a pre-trained model for a better inference performance, and also 
+analyze the model pb files before and after the inference optimizations.  
+
+| Optimized for                     | Description
+|:---                               |:---
+| OS                                | Linux* Ubuntu* 18.04 
+| Hardware                          | Intel® Xeon® Scalable processor family or newer
+| Software                          | Intel® oneAPI AI Analytics Toolkit
```

Comment:
I think it would be a good idea to put a link here to the AI toolkit

Reply:
put the link accordingly.